File: SDKs/googleapis.com/my-business/v4/java/lib/src/main/java/openapisdk/models/shared/LocationReview.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * LocationReview
 * Represents a review with location information.
**/
public class LocationReview {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("name")
    public String name;
    public LocationReview withName(String name) {
        this.name = name;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("review")
    public Review review;
    public LocationReview withReview(Review review) {
        this.review = review;
        return this;
    }
}

File: SDKs/googleapis.com/my-business/v4/python/src/sdk/models/shared/locationreview.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import review as shared_review


@dataclass_json
@dataclasses.dataclass
class LocationReview:
    r"""LocationReview
    Represents a review with location information.
    """
    
    name: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('name') }})
    review: Optional[shared_review.Review] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('review') }})
    
